gst/tensor_reposrc/tensor_reposrc.c

```diff
@@ -176,6 +178,18 @@ gst_tensor_reposrc_getcaps (GstBaseSrc * src, GstCaps * filter)
       cap = gst_caps_new_any ();
   }
 
+  st = gst_caps_get_structure (cap, 0);
+
+  name = gst_structure_get_name (st);
+
+  if (!g_str_equal (name, "other/tensor")
+      && !g_str_equal (name, "other/tensors")) {
+    GST_ELEMENT_ERROR (GST_ELEMENT (self), STREAM, WRONG_TYPE,
+        ("Only Tensor/Tensors MIME are supported for now"), (NULL));
+  }
+
+  gst_tensors_config_from_structure (&self->config, st);
```

Comment:
line 183 ~ 189, how about checking returned value of gst_tensors_config_from_structure,

gst_tensors_config_from_structure returns FALSE if ```GstStructure``` does not have name other/tensor or other/tensors.

```
  if (!gst_tensors_config_from_structure (&self->config, st)) {
    GST_ELEMENT_ERROR (GST_ELEMENT (self), STREAM, WRONG_TYPE,
        ("Only Tensor/Tensors MIME are supported for now"), (NULL));
}
```